src/extract_gms.py

```diff
@@ -362,15 +362,23 @@ def gms_extract(filename,states,min_shift,active_space,flag):
     # the total energies of excited states (1-electron basis)
 
     # *********Here, KS excitation energy -> total excitation energy***************
+    homo = active_space[0] - min_shift # HOMO : number ordering starts from 1, not 0.
```

Comment:
make sure this is the same convention that is used in Libra-QE as well, we want to keep the meaning of the "active_space" the same for all types of the interface